### PR TITLE
Added Ares Network and WiBn

### PR DESCRIPTION
**Pull request by HaeImAlan**
### General:
* [ ] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated servers.json`)*
* [ ] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [ ] No changes were made to the file's formatting.
* [ ] There are no syntax errors.
* [ ] There is no pre-existing mapping matching my id.
* [ ] My field values match your requirements.
* You can view our patterns here: [servers.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/servers.schema.json), or take a look below and complete the field checklist:
  - [ ] `id`: a lowercase string *(ex. `myserver`)*
  - [ ] `name`: a string *(ex. `MyServerPvP`)*
  - [ ] `addresses`: an array with lowercase strings *(ex. of address `my.server`)*
    - You do not need to specify sub-domains, Lunar Client services automatically detect them.

### Logo Additions or Updates:
* [ ] My image is a `png` file.
* [ ] I uploaded my image to the `logos` directory.
* [ ] My image has a transparent background and is square (1:1 aspect ratio).
* [ ] My image is `512` pixels in width and height.
* [ ] My image's file name matches its mapping id.

### CODE REVIEW

File: servers.json [hunk 1]
```diff
@@ -1411,5 +1411,19 @@
         "addresses": [
             "ymzc.pro"
         ]
+    },
+    {
+        "id": "wibn",
+        "name": "WiBn,
```
Comment (jacobk999) [inline servers.json:1417]: Fix your json formatting and upload your server logos

**Review by jacobk999 — CHANGES_REQUESTED** [review]

File: servers.json (hunk 1 above)
Comment (HaeImAlan) [reply servers.json]: Added the server logo and _hopefully_ fixed the json formatting

**Review by jacobk999 — APPROVED** [review]

**Review by jacobk999 — CHANGES_REQUESTED** [review]
Sorry for the late response, make sure your server logos are at least 512px by 512px

File: servers.json (hunk 1 above)
Comment (HaeImAlan) [reply servers.json]: Pretty sure the stuffs work now.

**Comment by HaeImAlan** [comment]
@jacobk999 I've fixed the server icons, not sure why it still says servers.json is conficting. Please take a look.

**Comment by jacobk999** [comment]
Why did the logos for servers other than Ares and WiBn change?

**Comment by HaeImAlan** [comment]
actually should i just close this pr and create a new one

**Pull request closed without merging**